docs/conf.py

```diff
@@ -194,7 +194,7 @@
 html_show_sphinx = False
 
 # If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
-html_show_copyright = True
+html_show_copyright = False
```

Comment:
This line, when set to "True", was wrapping another copyright symbol and period around the text in line 68, so I believe setting it to "False" removes that.